recipes/nslsii/meta.yaml

```diff
@@ -0,0 +1,76 @@
+{% set name = "nslsii" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c409688657771291fb2c32e95045d3f44432402ebfb41dae5bc7188c1d34c075
+  patches:
+    - 0001-import-bluesky_kafka-only-when-needed.patch
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  skip: true  # [py<37]
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - appdirs
+    - bluesky
+    - bluesky-kafka  # [not win]
+    - caproto
+    - databroker
+    - h5py
+    - ipython
+    - ipywidgets
+    - matplotlib
+    - numpy
+    - ophyd
+    - psutil
+    - pyolog
+
+test:
+  requires:
+    - pip
+  commands:
+    - pip check
```

Comment:
I suggest removing it as `nslsii 0.1.3 requires bluesky-kafka, which is not installed.`

We skip this dependency here, but not in https://github.com/NSLS-II/nslsii/blob/904e15e77f2174c3d2ff47b7b7984db0bd360bb2/requirements.txt#L3.

Reply:
@xhochy, are you OK with that?

Reply:
Actually, I think these 4 lines can be added conditionally for non-Windows builds.

The same is true for `hxntools` that is failing with the same issue:
```
%SRC_DIR%>conda activate "%PREFIX%" 
import: 'hxntools'
import: 'hxntools'
nslsii 0.1.3 requires bluesky-kafka, which is not installed.
TESTS FAILED: hxntools-0.5.1-pyhd8ed1ab_0.tar.bz2
```